### DISCUSSION TITLE
[Feature] Duplicates desktop scaling

### DISCUSSION THREAD

**Discussion by Hantick** [immich-app/immich#10548] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

On 2K resolution this section looks kinda small:
![image](https://github.com/immich-app/immich/assets/36815427/02524350-568b-49d8-9f62-f644906899a2)
How it could look like:
![image](https://github.com/immich-app/immich/assets/36815427/2bfe5480-1d36-4b52-9675-c542fb00a764)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by mirisbowring** [comment]
Yep, it would be greate to be able to click on a image or have an side by side comparison in full resolution. Sometime i have multiple photos on low light and most of them are a bit blurry that is only visible on a large photo (not on a thumbnail) currently I am not able to determine which of them is the sharper one

**Comment by RalfKo** [comment]
I use a 5K monitor, To me this is a VERY obvious feature request.
